TINY-9827: Move the cursor out of the accordion at the document edges (#8985)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

- In some cases pressing enter would scroll the entire page. #TINY-9828
- The border styles of a table were incorrectly split into a longhand form after table dialog updates. #TINY-9843
- Links in `help dialog` -> `plugins` and `version` were not navigable via keyboard. #TINY-10071

> - Fixed the inability to insert content before the `details` element, when it is the first element in the editor's content. Now possible by pressing the `Up` arrow key when the cursor is at the start of the `details` element content. #TINY-9827

Reviewer: We should mention that both up and down work - also perhaps that up was added for tables?

It's hard to say in a single line, maybe we need 2 (or 3 with the table change).

```suggestion
- Fixed the inability to insert content next to the `details` element when it is the first or last content element. Now pressing the `Up` or `Down` arrow key will insert a block element before or after the `details` element. #TINY-9827
```
